pkg/kubelet/dockershim/network/plugins.go

```diff
@@ -242,16 +242,23 @@ func getOnePodIP(execer utilexec.Interface, nsenterPath, netnsPath, interfaceNam
 	if len(lines) < 1 {
 		return nil, fmt.Errorf("unexpected command output %s", output)
 	}
-	fields := strings.Fields(lines[0])
-	if len(fields) < 4 {
-		return nil, fmt.Errorf("unexpected address output %s ", lines[0])
-	}
-	ip, _, err := net.ParseCIDR(fields[3])
-	if err != nil {
-		return nil, fmt.Errorf("CNI failed to parse ip from output %s due to %v", output, err)
+	for  _, line := range lines {
+		fields := strings.Fields(line)
+		if len(fields) < 6 {
+			continue
+		}
+		if flieds[5] != "global" && flieds[5] != "site" {
```

Comment:
Nit: I'm fairly confident tests are going to fail bc of a typo here :) (i.e. `s/flieds/fields/`).